pkg/nseconfig/config.go

```diff
@@ -13,14 +13,20 @@ type Config struct {
 }
 
 type Endpoint struct {
-	Name   string `yaml:"name"`
-	Labels Labels `yaml:"labels"`
-
-	NseName string `yaml:"nseName"` //TODO temporary in order to be able to run examples
-
-	NseControl *NseControl `yaml:"nseControl"`
+	Name   		string `yaml:"name"`
+	Labels 		Labels `yaml:"labels"`
+	NseName 	string `yaml:"nseName"` //TODO temporary in order to be able to run examples
+	NseControl 	*NseControl `yaml:"nseControl"`
+	VL3 		VL3 `yaml:"vl3"`
+	AwsTgwEndpoint *AwsTgwEndpoint
```

Comment:
Format this file. Also you could add after ``AwsTgwEndpoint *AwsTgwEndpoint`` **\`yaml:"awsTgwEndpoint"\`**

Reply:
Sure, what should I format exactly? I am moving the annotations so that they all start in the same column. Should I do anything else?